src/main/java/com/iota/iri/service/dto/GetNeighborsResponse.java

```diff
@@ -42,8 +42,10 @@ public String getConnectionType() {
 
         public static Neighbor createFrom(com.iota.iri.network.Neighbor n) {
             Neighbor ne = new Neighbor();
+            java.net.InetSocketAddress socketAddress = n.getAddress();
```

Comment:
could you please use an `import` statement instead of using the fully qualified name?

Reply:
@cwarner818 please add `import java.net.InetSocketAddress;` at the head of the file and change line 45 from `java.net.InetSocketAddress socketAddress = n.getAddress();` to `InetSocketAddress socketAddress = n.getAddress();`